descholar-ceo/AnnounceIT
# #170727979 Users sees all announcements

#### What does this PR ​ do​ ?
As an admin, I should be able to view all announcements from all users

#### Description of Task to be completed?
* Admin clicks on __All announcements__ tab or menu, 
* He/she get redirected to the page containing all announcement from all users

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/admin-side/all-announcements.html`, will be redirected to the page which contains all announcements, or enter like this `http://127.0.0.1:5500/UI/html/admin-side/all-announcements.html`

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/admin-side/` and you will see different files, you choose `all-announcements.html` and right click and open with your preferred browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170727979](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-19 15-31-40](https://user-images.githubusercontent.com/56886064/72681915-d89d0880-3ad0-11ea-8d14-da7b14ed33c1.png)

#### Questions:
No Question
